Admin/AbstractAdminDefinition.php

```diff
@@ -40,6 +40,13 @@ public function getExportFormats()
     /**
      * {@inheritdoc}
      */
+    public function configureExportFields(&$fields)
```

Comment:
We already used php references in some others methods. So, this should be no problem at all.